pkg/controller/memberstatus/memberstatus_controller.go

```diff
@@ -291,6 +296,47 @@ func (r *ReconcileMemberStatus) routesHandleStatus(reqLogger logr.Logger, member
 	return nil
 }
 
+// cheIntegrationHandleStatus checks all necessary aspects related integration between the member operator and Che
+// Returns an error if any problems are discovered.
+func (r *ReconcileMemberStatus) cheIntegrationHandleStatus(reqLogger logr.Logger, memberStatus *toolchainv1alpha1.MemberStatus) error {
+	if memberStatus.Status.Che == nil {
+		memberStatus.Status.Che = &toolchainv1alpha1.CheStatus{}
+	}
+
+	// Is che user deletion enabled
+	if !r.config.IsCheUserDeletionEnabled() {
+		// Che user deletion is not enabled, set condition to Ready. No further checks required
+		readyCondition := status.NewComponentReadyCondition(toolchainv1alpha1.ToolchainStatusMemberStatusCheUserDeletionNotEnabledReason)
+		memberStatus.Status.Che.Conditions = []toolchainv1alpha1.Condition{*readyCondition}
+		return nil
+	}
+
+	if r.config.GetCheAdminUsername() == "" || r.config.GetCheAdminPassword() == "" {
```

Comment:
You intended to use `isCheAdminUserConfigured()` instead, right?

Reply:
Yes, I refactored this stuff too much and lost track. 🙂  I was originally using it in the useraccount_controller too so it was a common function but then I created the `IsCheUserDeletionEnabled` function and used it there instead. This function is probably not necessary now but I'll still go ahead and use it because it makes the condition slightly more understandable. Done in 5fe5a24cdf688f31af552f325423ac3b5cc77129